docs/contributing/development/README.md

```diff
@@ -0,0 +1,50 @@
+# Development Guide
+
+We're happy you're interested in contributing to the Tetragon project.
+
+This section of the Tetragon documentation will help you make sure you
+have an environment capable of testing changes to the Tetragon source code,
+and that you understand the workflow of getting these changes reviewed and
+merged upstream.
+
+## How To Contribute
+
+### Clone and Provision Environment
+
+1. Make sure you have a [GitHub account](https://github.com/join)
+
+2. Fork the Tetragon repository to your GitHub user or organization.
+
+3. Turn off GitHub actions for your fork as described in the [GitHub
+Docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#managing-github-actions-permissions-for-your-repository>)
+   - This is recommended to avoid unnecessary CI notification failures on the fork.
+
+4. Clone your `${YOUR_GITHUB_USERNAME_OR_ORG}/tetragon` fork into your `GOPATH`, and setup the base repository as `upstream` remote:
+
+```
+   mkdir -p "${GOPATH}/src/github.com/cilium"
+   cd "${GOPATH}/src/github.com/cilium"
+   git clone https://github.com/${YOUR_GITHUB_USERNAME_OR_ORG}/tetragon.git
+   cd tetragon
+   git remote add upstream https://github.com/cilium/tetragon.git
+```
+
+5. Prepare your [Development Setup](#development-setup), see below.
+
+6. Check the GitHub issues for [good tasks to get
+started](https://github.com/cilium/tetragon/issues?q=is%3Aopen+is%3Aissue+label%3Agood-first-issue)
+
+
+### Developer’s Certificate of Origin
+
+To improve tracking of who did what, we’ve introduced a “sign-off” procedure,
+make sure to read and apply the 
+[Developer’s Certificate of
+Origin](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#developer-s-certificate-of-origin).
+
+
+## Development Setup
+
+Documentation for Development setup is [here][develop].
+
+[develop]: ./DEVELOP.md
```

Comment:
WDYT about folding DEVELOP.md into here as a flat section? I'd rather not have it buried behind two clicks.

Reply:
@willfindlay sure, done.